src/UI/walletConnect/WalletConnectLoginContainer/index.tsx

```diff
@@ -43,12 +47,31 @@ export const WalletConnectLoginContainer = ({
     logoutRoute,
     callbackRoute,
     token,
-    shouldLoginUser: true,
+    onLoginRedirect
+  });
+  const [
+    initLoginWithWalletConnectV2,
+    connectExisting,
+    { error: walletConnectErrorV2 },
+    {
+      uriDeepLink: walletConnectDeepLinkV2,
+      walletConnectUri: walletConnectUriV2,
+      wcPairings
+    }
+  ] = useWalletConnectV2Login({
+    logoutRoute,
+    callbackRoute,
+    token,
     onLoginRedirect
   });
   const [qrCodeSvg, setQrCodeSvg] = useState<string>('');
   const isMobileDevice =
     platform?.os?.family === 'iOS' || platform?.os?.family === 'Android';
+  const activePairings =
+    isWalletConnectV2 && wcPairings && wcPairings?.length > 0
```

Comment:
I think that `&& wcPairings` can be deleted since you're using optional chaining

Reply:
needed for typescript for now. Without it we get the `object is possibly undefined ` even though the code wouldn't actually run that case